test/lib/incentives-calculation.test.ts

```diff
@@ -216,9 +214,9 @@ test('correctly evaluates scenario "Single w/ $120k Household income in NJ"', as
     i => i.payment_methods[0] === PaymentMethod.PerformanceRebate,
   );
 
-  t.equal(pos_rebate_incentives.length, 7);
-  t.equal(tax_credit_incentives.length, 12);
-  t.equal(performance_rebate_incentives.length, 1);
+  t.equal(pos_rebate_incentives.length, 0);
+  t.equal(tax_credit_incentives.length, 10);
```

Comment:
The number went down because the "state" calculation function doesn't return ineligible incentives at all, rather than returning them with `eligible: false`. The end result in the v1 API is the same.